resources/js/components/fieldtypes/markdown/MarkdownFieldtype.vue

```diff
@@ -1,120 +1,133 @@
-                    </div>
-                    <div class="markdown-buttons" v-if="! isReadOnly">
-                        <button
-                            v-for="button in buttons"
-                            v-tooltip="button.text"
-                            :aria-label="button.text"
-                            @click="button.command(editor)"
-                        >
-                            <svg-icon :name="button.svg" class="w-4 h-4" />
-                        </button>
-                        <button @click="toggleDarkMode" v-tooltip="darkMode ? __('Light Mode') : __('Dark Mode')" :aria-label="__('Toggle Dark Mode')" v-if="fullScreenMode">
-                            <svg-icon name="dark-mode" class="w-4 h-4" />
-                        </button>
-                        <button @click="toggleFullScreen" v-tooltip="__('Toggle Fullscreen')" :aria-label="__('Toggle Fullscreen Mode')">
-                            <svg-icon name="expand-bold" class="w-4 h-4" v-show="!fullScreenMode" />
-                            <svg-icon name="arrows-shrink" class="w-4 h-4" v-show="fullScreenMode" />
-                        </button>
-                    </div>
-                </div>
-
-                <div class="drag-notification" v-show="dragging">
-                    <svg-icon name="upload" class="h-12 w-12 mb-4" />
-                    {{ __('Drop File to Upload') }}
-                </div>
-
-                <uploads
-                    v-if="uploads.length"
-                    :uploads="uploads"
-                    class="-mt-px"
-                />
-
-                <div :class="`mode-wrap mode-${mode}`" @click="focus">
-                    <div class="markdown-writer"
-                        ref="writer"
-                        v-show="mode == 'write'"
-                        @dragover="draggingFile = true"
-                        @dragleave="draggingFile = false"
-                        @drop="draggingFile = false"
-                        @keydown="shortcut">
-
-                        <div class="editor" ref="codemirror"></div>
-
-                        <div class="helpers">
-                            <div class="flex w-full">
-                                <div class="markdown-cheatsheet-helper">
-                                    <button class="text-link flex items-center" @click="showCheatsheet = true" :aria-label="__('Show Markdown Cheatsheet')">
-                                        <svg-icon name="markdown-icon" class="w-6 h-4 items-start rtl:ml-2 ltr:mr-2" />
-                                        <span>{{ __('Markdown Cheatsheet') }}</span>
-                                    </button>
-                                </div>
+    <portal name="markdown-fullscreen" :disabled="!fullScreenMode" target-class="markdown-fieldtype">
```

Comment:
Can you please undo the formatting of this file? It'll make it easier for us to see just what's changed. 😄

Reply:
I got it. 👍